frontend/src/components/common/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC } from 'common/types/types';
+import { Icon } from '../icon';
+import { IconName } from 'common/enums/components';
+import { SubscribeChanel } from './sidebar-container';
+import { Chanel } from 'components/sidebar/chanel';
+import { Link } from '../common';
+import { AppRoute } from 'common/enums/enums';
+
+import styles from './sidebar.module.scss';
+
+interface SidebarProps {
+  subscribesList: SubscribeChanel[];
+}
+
+type ActiveClass = {
+  isActive: boolean;
+};
+
+const returnActiveClass = ({ isActive }: ActiveClass): string => (isActive ? styles['active'] : '');
+
+const Sidebar: FC<SidebarProps> = ({ subscribesList }) => (
+  <div className={styles.sidebar}>
+    <nav className={styles['navigate-menu']}>
+      <ul>
+        <Link to={AppRoute.ROOT} className={returnActiveClass}>
+          <li>
+            <Icon name={IconName.HOME} width="24" height="24" />
+            <span className={styles['link-name']}>Home</span>
+          </li>
+        </Link>
+        <Link to={AppRoute.BROWSE} className={returnActiveClass}>
+          <li>
+            <Icon name={IconName.COMPAS} width="24" height="24" />
+            <span className={styles['link-name']}>Browse</span>
+          </li>
+        </Link>
+        <Link to={AppRoute.FOLLOWING} className={returnActiveClass}>
+          <li>
+            <Icon name={IconName.FOLLOW} width="24" height="24" />
+            <span className={styles['link-name']}>Following</span>
+          </li>
+        </Link>
+        <Link to={AppRoute.HISTORY} className={returnActiveClass}>
+          <li>
+            <Icon name={IconName.TIMEAGO} width="24" height="24" />
+            <span className={styles['link-name']}>History</span>
+          </li>
+        </Link>
```

Comment:
You are duplicating same construction each time. You can move that to config like this:

```javascript
.... {
  iconName: Icon.COMPAS,
  text: "Browse",
  linkTo: AppRoute.BROWSE,
}
```

Then create a function for checking if it's active by routeName or id